style.css

```diff
@@ -0,0 +1,27 @@
+ .display{ 

```

Comment:
give spaces between the selector name and the curly brace for the declaration: instead of `.display{` you should have `.display {`